packages/util/src/use-unique-id.ts

```diff
@@ -0,0 +1,13 @@
+import { useLayoutEffect, useState } from 'react'
+
+import { uniqueId } from './unique-id'
+
+export function useUniqueId(prefix = '') {
+  const [id, setId] = useState('')
+
+  useLayoutEffect(() => {
```

Comment:
Why does this have to be useLayoutEffect instead of useEffect?

Reply:
i actually don't remember...i just copied this over